content/docs/command-reference/exp/apply.md

```diff
@@ -0,0 +1,138 @@
+# exp apply
+
+Apply the changes from a previously run experiment into the workspace.
+
+## Synopsis
+
+```usage
+usage: dvc exp apply [-h] [-q | -v] experiment
+
+positional arguments:
+  experiment     Experiment to be applied.
+```
+
+## Description
+
+This command will apply the changes from an experiment to the workspace. It is
+usually run when you are ready to Git commit the results of an experiment into
+the project.
+
+This command will fail if the specified experiment was not derived from the Git
+commit which is currently (Git) checkout-ed in the workspace.
+
+## Options
+
+- `experiment` - name or revision of the experiment to be checked out.
```

Comment:
No need to include here for now unless there's very complex logic about this argument. We will revisit this question later for all refs in #2024
```suggestion
```

Reply:
We should explain the ways to refer to experiments (name or revision) somewhere. If that's in the Description then this isn't needed (I incline for that), otherwise it can be here.